README.md

```diff
@@ -279,6 +322,32 @@ can test against a different image/tag like so:
 
   `$ MOLECULE_DISTRO="fedora:30" molecule test`
 
+CI tests
+========
+In the tests directory, currently the following test scripts are available.
+tests_combination.yml    tests_default.yml              tests_files_files.yml              tests_files.yml
+tests_combination2.yml   tests_enabled.yml              tests_files_forwards_implicit.yml  tests_forwards.yml
+tests_default_files.yml  tests_files_elasticsearch.yml  tests_files_forwards.yml           tests_listen.yml
```

Comment:
You can omit the list of tests, and just say "The tests are `tests/tests*.yml`" since this list may change in the future and we don't want to have to keep this doc in sync.